### PR TITLE
fix: various typos fixed

### LINKED ISSUE #55275: Various typos
One "not" should be removed:
https://github.com/freeCodeCamp/freeCodeCamp/blob/e5a5a308faebe00eb72353de172103dded6f90a9/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da4b16937be43ba24c63d.md?plain=1#L59

---

It should be variable, singular:
https://github.com/freeCodeCamp/freeCodeCamp/blob/e5a5a308faebe00eb72353de172103dded6f90a9/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d39c156fe94b7482c3ab6.md?plain=1#L26

---

It should be "its", without the apostrophe:
https://github.com/freeCodeCamp/freeCodeCamp/blob/e5a5a308faebe00eb72353de172103dded6f90a9/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d4813c17b37d1e261a566.md?plain=1#L10

### PR DESCRIPTION
**Pull request by rohan220217**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55275

### CODE REVIEW

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please do as the bot says, remove the changes from non-english files

**Comment by ilenia-magoni** [comment]
> Closes #55275

if your PR target multiple issues, you need to add that it closes the other issues too. But please solve only one issue.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da4b16937be43ba24c63d.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Your second `.then()` method should log `data` to the console.
 assert.match(code, /fetch\(\s*('|"|`)https:\/\/cdn\.freecodecamp\.org\/curriculum\/news\-author\-page\/authors\.json\1\s*\)\s*\.then\(\s*(\(\s*res\s*\)|res)\s*=>\s*res\.json\(\s*\)\s*\)\s*\.then\(\s*(\(\s*data\s*\)|data)\s*=>\s*\{\s*\n?\s*?console\.log\(\s*data\s*\)\s*;?\s*\}\s*\)/)
 ```
 
-Your second `.then()` should not not have semi-colon after it.
+Your second `.then()` should not have semi-colon after it.
```
Comment (lasjorg) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641da4b16937be43ba24c63d.md:59]:
```suggestion
Your second `.then()` should not have a semicolon after it.
```

**Review by lasjorg — CHANGES_REQUESTED** [review]

**Comment by lasjorg** [comment]
As said, please remove the change for the binary typo. It is handled in another PR and does not belong in this PR.

**Review by JoyShaheb — APPROVED** [review]

**Comment by gikf** [comment]
Hey @rohan220217

Thank you for opening this pull request.

As there was different PR - #55296 - making all necessary changes, merged recently, we are going to close this one.

Thank you and happy coding! 😊

---

If you have any questions, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [the contributors chat room](https://discord.gg/PRyKn3Vbay).

**Pull request closed without merging**